components/header.js

```diff
@@ -1,34 +1,72 @@
-import Link from "next/link"
+import Link from "next/link";
+import styles from "./style.module.css";
+import Image from "next/image";
+
+import React, { useEffect, useState } from "react";
 
 export default function Header() {
-    
-  
-    return (
-      <header>
-        <nav>
-          <ul className="">
-            <li className="">
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    fetch("http://turismo:8081/rest/weather")
+      .then((response) => response.json())
+      .then((json) => setItems(json))
+      .catch((error) => alert("Error" + error.message));
+  }, []);
+  const weather = {
+    main: items.main,
+    icon: items.icon,
+    celsius : items.celsius,
+  };
+
+  return (
+    <header>
+      <div className={styles.cabecera}>
+        <nav className={styles.navegator}>
+          <ul className={styles.listaNav}>
+            <li className={styles.liNav}>
               <Link href="/">
                 <a>Home</a>
               </Link>
             </li>
-            <li className="">
-              <Link href="/client">
+            <li className={styles.liNav}>
+              <Link href="/restaurants">
                 <a>Restaurant</a>
               </Link>
             </li>
-            <li className="">
-              <Link href="/server">
+            <li className={styles.liNav}>
+              <Link href="/gasStations">
                 <a>GasStations</a>
               </Link>
             </li>
-            <li className="">
-              <Link href="/protected">
+            <li className={styles.liNav}>
+              <Link href="/videos">
+                <a>Videos</a>
+              </Link>
+            </li>
+            <li className={styles.liNav}>
+              <Link href="/map">
                 <a>Map</a>
               </Link>
             </li>
           </ul>
         </nav>
-      </header>
-    )
-  }
\ No newline at end of file
+        <div className={styles.ventanaClima}>
+          <h1 className={styles.tituloCelsius}>{weather["celsius"]} C</h1> 
+          <div className={{ width: "30%", height: "30%" }}>
+            <img
+              src={
+                "http://openweathermap.org/img/wn/" +
```

Comment:
this is a kind of conditional stuff, so please create an internal function to get the value and put the url on a CONSTANT file "http://openweathermap.org/img/wn/"

Reply:
i changed that and put the correct form. thanks for all